baseframe/templates/baseframe/mui/forms.html.jinja2

```diff
@@ -38,8 +37,9 @@
         <div>
           <div class="mui-checkbox">
             <label>
-              {{ rendererrors(field) }}
               {{ field | render_field_options(class="field-" + field.id + " " + widget_css_class, tabindex=tabindex, rows=rows) }} {{ field.label.text }}
+              {{ rendererrors(field) }}
+              {%- if error_template %}<p class="mui-form--error" id="{{field.name}}-error"></p>{%- endif %}
```

Comment:
Shouldn't the `rendererrors` macro do this? Is `error_template` a parameter to be passed only for the MUI theme?

Reply:
Since the form is rendered client side on hgtv spa, this empty p field was added so that client can render the field errors just below it.

Reply:
I'm not comfortable with this. This is a render flag that has to be set in business logic. We can't be passing around an `error_template` parameter. The code that displays errors has to use some other mechanism to find the right place to display them.